sr_multi_moveit/right_sr_ur10_moveit_config/test/test_planners.py

```diff
@@ -0,0 +1,166 @@
+        
+        #Visualize the goal position
+        #self.goal_pub = rospy.Publisher('tested_goal', DisplayRobotState, queue_size=10)
+        #self.msg = DisplayRobotState()
+        #self.msg.state.joint_state.name =  ['ra_shoulder_pan_joint', 'ra_shoulder_lift_joint', 'ra_elbow_joint', 'ra_wrist_1_joint', 'ra_wrist_2_joint', 'ra_wrist_3_joint', 'rh_WRJ2', 'rh_WRJ1', 'rh_FFJ4', 'rh_FFJ3', 'rh_FFJ2', 'rh_FFJ1', 'rh_MFJ4', 'rh_MFJ3', 'rh_MFJ2', 'rh_MFJ1', 'rh_RFJ4', 'rh_RFJ3', 'rh_RFJ2', 'rh_RFJ1', 'rh_LFJ5', 'rh_LFJ4', 'rh_LFJ3', 'rh_LFJ2', 'rh_LFJ1', 'rh_THJ5', 'rh_THJ4', 'rh_THJ3', 'rh_THJ2', 'rh_THJ1']
+        #self.msg.state.joint_state.position = [1.57542451065, 3.01734161219, 2.01043257686, -1.14647092839, 0.694689321451, -0.390769365032, 0.0, 0.0, 0.0, 0.0, 0.0, 0.349938515025, 0.0, 0.0, 0.0, 0.349938515025, 0.0, 0.0, 0.0, 0.349938515025, 0.0, 0.0, 0.0, 0.0, 0.349938515025, 0.0, 0.0, 0.0, 0.0, 0.349938515025]
+        #rospy.sleep(5)
+           
+    def _add_walls_and_ground(self):       
+        # publish a scene
+        p = geometry_msgs.msg.PoseStamped()
+        p.header.frame_id = self.robot.get_planning_frame()
+        
+        p.pose.position.x = 0
+        p.pose.position.y = 0
+        # offset such that the box is below ground (to prevent collision with the robot itself)
+        p.pose.position.z = -0.11
+        p.pose.orientation.x = 0
+        p.pose.orientation.y = 0
+        p.pose.orientation.z = 0
+        p.pose.orientation.w = 1
+        self.scene.add_box("ground", p, (3, 3, 0.1))
+        
+        p.pose.position.x = 0.4
+        p.pose.position.y = 0.85
+        p.pose.position.z = 0.4
+        p.pose.orientation.x = 0.5
+        p.pose.orientation.y = -0.5
+        p.pose.orientation.z = 0.5
+        p.pose.orientation.w = 0.5
+        self.scene.add_box("wall_front", p, (0.8, 2, 0.01))
+        
+        p.pose.position.x = 1.33
+        p.pose.position.y = 0.4
+        p.pose.position.z = 0.4
+        p.pose.orientation.x = 0.0
+        p.pose.orientation.y = -0.707388
+        p.pose.orientation.z = 0.0
+        p.pose.orientation.w = 0.706825
+        self.scene.add_box("wall_right", p, (0.8, 2, 0.01))
+        
+        p.pose.position.x = -0.5
+        p.pose.position.y = 0.4
+        p.pose.position.z = 0.4
+        p.pose.orientation.x = 0.0
+        p.pose.orientation.y = -0.707107
+        p.pose.orientation.z = 0.0
+        p.pose.orientation.w = 0.707107
+        self.scene.add_box("wall_left", p, (0.8, 2, 0.01))
+        
+        #rospy.sleep(1)
+          
+    def _check_plan(self, plan):
+        if len(plan.joint_trajectory.points) > 0:
+            #rospy.sleep(5)
+            return True
+        else:
+            return False
+       
+    def _plan_joints(self, joints):
+         self.group.clear_pose_targets()
+         group_variable_values = self.group.get_current_joint_values()
+         group_variable_values[0:6] = joints[0:6]
```

Comment:
Bah anyway you have to use the array of fixed joint positions in the test_trajectories below so leave it like this :)